cmd/csppserver/http.go

```diff
@@ -178,6 +180,22 @@ CoinShuffle++ change.</p>
 imperative that addresses are never reused and that extended public keys of
 mixed and voting accounts are not revealed to other parties.</p>
 
+{{ if .OnionServerName }}
+<h2>Connecting to CSPP server via Tor</h2>
```

Comment:
Let's name this header "Tor Hidden Service".  The public server can already be reached over tor.